api/v1alpha1/nfspvc_webhook.go

```diff
@@ -102,7 +104,7 @@ func (r *NfsPvc) validateAccessMode(accessMode []corev1.PersistentVolumeAccessMo
 func (r *NfsPvc) doesPVCExist(K8sClient client.Client) bool {
 	pvc := corev1.PersistentVolumeClaim{}
 	if err := K8sClient.Get(context.Background(), types.NamespacedName{Namespace: r.Namespace, Name: r.Name}, &pvc); err != nil {
-		if errors.IsNotFound(err) {
+		if apierrors.IsNotFound(err) {
```

Comment:
I'll change the name to k8sErrors

Reply:
I think we have other code which has it as `apierrors` but I don't feel very strongly about it so I'll make the change